tests/unit/test_modulegraph/test_modulegraph.py

```diff
@@ -290,6 +295,38 @@ def test_find_module(self):
                 self.assertEqual(description, (ext, 'rb', imp.C_EXTENSION))
                 self.assertEqual(fp, None)
 
+            # Module in Package
+            parent = MockNode('mypkg')
+            info = modgraph._find_module('__init__', path=[path, os.path.join(path, 'mypkg')] + sys.path, parent=parent)
+            fp = info[0]
+            filename = info[1]
+            description = info[2]
+
+            self.assertTrue(hasattr(fp, 'read'))
+            self.assertEqual(filename, os.path.join(path, 'mypkg', '__init__.py'))
+            self.assertEqual(description, ('.py', READ_MODE, imp.PY_SOURCE))
+
+            # Extension in Package
+            if path.endswith('.zip') or path.endswith('.egg'):
+                pass
+            else:
+                parent = MockNode('myextpkg')
+                info = modgraph._find_module('__init__', path=[path, os.path.join(path, 'myextpkg')] + sys.path, parent=parent)
+                fp = info[0]
+                filename = info[1]
+                description = info[2]
+
+                if sys.platform == 'win32':
+                    ext = '.pyd'
+                else:
+                    # This is a ly, but is good enough for now
```

Comment:
What does this comment mean?

Reply:
I do not understand either.
It's same as https://github.com/pyinstaller/pyinstaller/blob/develop/tests/unit/test_modulegraph/test_modulegraph.py#L286